xdsl/dialects/x86/ops.py

```diff
@@ -557,15 +557,15 @@ class R_RM_Operation(
 
     r1 = operand_def(R1InvT)
     r2 = operand_def(R2InvT)
-    offset = opt_attr_def(IntegerAttr)
+    offset = attr_def(IntegerAttr)
```

Comment:
We actually have functionality to make the attribute have a default value if it's not defined. I would actually recommend doing it this way:
```suggestion
    offset = attr_def(IntegerAttr, default_value=IntegerAttr(0, 64))
```